build-support/scripts/up

```diff
@@ -30,7 +30,9 @@ echo "Connected!"
 
 # Apply limited license
 echo -n "Applying license.. "
-docker exec -e "NUODB_LIMITED_LICENSE_CONTENT=$NUODB_LIMITED_LICENSE_CONTENT" ad1 /bin/bash -c 'echo "$NUODB_LIMITED_LICENSE_CONTENT" | base64 -d > ~/nuodb.lic && nuocmd set license --license-file ~/nuodb.lic && rm ~/nuodb.lic'
+docker exec -e "NUODB_LIMITED_LICENSE_CONTENT=${NUODB_LIMITED_LICENSE_CONTENT}" ad1 /bin/bash -c 'echo "${NUODB_LIMITED_LICENSE_CONTENT}" | base64 -d > ~/nuodb.lic && nuocmd set license --license-file ~/nuodb.lic && rm ~/nuodb.lic'
+#docker cp /u/users/dwe5/license.txt ad1:/tmp
+#docker exec ad1 /opt/nuodb/bin/nuocmd set license --license-file /tmp/license.txt
```

Comment:
I prefer not to remove it because at the moment I need to use these lines instead of the expected docker line to add the license.  I can't figure out why I am having a problem, but these commands allowed me to work around it, so until I can find an answer, I thought I would keep these lines around to make my life easier.

Reply:
If you want something that works on your machine, then make it generally useful.

You can conditionalize usage of the `NUODB_LIMITED_LICENSE_CONTENT` environment variable and fall back to `NUODB_LIMITED_LICENSE` or `NUODB_LICENSE`, which can be a file supplied by the user:

```bash
if [ "$NUODB_LIMITED_LICENSE_CONTENT" != "" ]; then
    docker exec -e "NUODB_LIMITED_LICENSE_CONTENT=${NUODB_LIMITED_LICENSE_CONTENT}" ad1 /bin/bash -c 'echo "${NUODB_LIMITED_LICENSE_CONTENT}" | base64 -d > ~/nuodb.lic && nuocmd set license --license-file ~/nuodb.lic && rm ~/nuodb.lic'
elif [ "$NUODB_LIMITED_LICENSE" != "" ]; then
    docker cp "$NUODB_LIMITED_LICENSE" ad1:/tmp/nuodb.lic
    docker exec ad1 nuocmd set license --license-file /tmp/nuodb.lic
fi
```

Reply:
While I don't disagree, I am still pissed I can't understand why my environment is doing something unique, Eusebio does not suffer the problems I see when he uses the same ma machine, but I have yet to see what is different.  It is not my intention to keep this choice around, I just wanted the reminder in the file as a way not to forget.  Certainly I can make the change but it was not my intent to provide another option for any reason for others to follow.

Reply:
I was not able to remove my need for the second approach, I just committed and pushed something similar to what you suggested, just changed the check and double checked that at least one of the approaches is used.  Have a look to see if we can agree on approval to move this forward.